source/develop/release-management/features/virt/dedicated-cpus.md

```diff
@@ -0,0 +1,68 @@
+---
+Title: Dedicated CPUs
+Category: feature
+Authors: sabusale
+---
+
+# Dedicated CPUs
+
+## Summary
+
+Once choosing dedicated CPU policy, The guest vCPUs will be strictly pinned to a set of host pCPUs (similarly to static CPU pinning). The set of pCPUs will be chosen to match the required guest CPU topology.
+
+## Owner
+
+*   Name: Saif Abu Saleh (sabusale)
+*   Email: sabusale@redhat.com
+
+
+### Tasks:
+
+
+1- Add dedicated to CPU Pinning Policy enum
+
+2- Scheduling
+
+3- Send vcpupin with cpusets and policy name in VM metadata(only when Policy is dedicated?)
+
+4- Send the policy name from vmStatic policy field
+
+5- Send the cpusets (convert from CurrentCpuPinningField form to array)
+
+6- API: Migrate
```

Comment:
On migration you will need to re-do steps 2-5. You need to find a host that has enough CPUs to use, and then you need to re-create the pinning (as it is possible that the new host freed CPUs are differ from the current used ones).

Reply:
why it need to be done in the migration? migration should call the scheduler to handle this right?

Reply:
Answered on later discussion.